modules/apps/object/object-test/src/testFunctional/macros/ObjectAdmin.macro

```diff
@@ -625,6 +626,26 @@ definition {
 			value1 = "Required");
 	}
 
+	macro assertStandaloneUpdateEntryError {
```

Comment:
You can remove this macro. BPM team is moving away from assertions at the macro level and need any assertions to be made on the testcase level. These steps can be added to the test itself.